infrastructure/zk/src/hyperchain_wizard.ts

```diff
@@ -630,6 +630,19 @@ export function getTokens(network: string): L1Token[] {
     }
 }
 
+export function getNativeToken(): L1Token {
+    const configPath = `${process.env.ZKSYNC_HOME}/etc/tokens/native_erc20.json`;
+    try {
+        return JSON.parse(
+            fs.readFileSync(configPath, {
+                encoding: 'utf-8'
+            })
+        );
+    } catch (e) {
+        throw e;
+    }
```

Comment:
Let me know if I'm overlooking smth, but if you plan to leave the error as is and simply propagate it, can you remove the `try/catch` block and keep only the parsing function, right?

Reply:
Correct, I'm still confused about some aspects of the language.